bats/.gitignore

```diff
@@ -0,0 +1 @@
+helper/__pycache__
```

Comment:
running python from bats creates this __pycache__ directory which we don't want checked in.